foundry/hooks.d.ts

```diff
@@ -740,12 +747,30 @@ declare namespace Hooks {
     userId: number
   ) => void;
 
-  /**
-   * This is called when the official world time changes.
-   * @param worldTime - the new world time
-   * @param dt        - the time advance delta, in seconds
-   * @remarks The name is 'updateWorldTime'.
-   * @see {@link GameTime#onUpdateWorldTime}
-   */
-  type UpdateWorldTime = (worldTime: number, dt: number) => void;
+  type DynamicCallbacks =
+    | CloseApplication
+    | ControlPlaceableObject
+    | CreateEmbeddedEntity
+    | CreateEntity
+    | DeleteEmbeddedEntity
```

Comment:
This one is typed incorrectly, I think. It's typed to receive an `Entity`, but that's not true, it's the data instead. Not all embedded entities are `Entity`s anyways.